public/js/components/AtomEdit/CustomEditors/CTAEditor.js

```diff
@@ -18,19 +18,19 @@ export class CTAEditor extends React.Component {
     return (
       <div className="editor editor-cta">
         <ManagedForm data={this.props.atom} updateData={this.props.onUpdate}>
-          <ManagedField fieldLocation="data.url" name="Link Url" isRequired={true}>
+          <ManagedField fieldLocation="data.url" name="Link Url" fieldId="url" isRequired={true}>
```

Comment:
@clloyd Hmm, looks like for now, the key is being set to `.0` for all of them, so it's not unique. This was more a change to fix the spaces in the `id`